src/services/user.service.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-return-await */
+
+const { User } = require('../models/User.model');
+
+const getAll = async () => {
+  const result = await User.findAll();
+
+  return result;
+};
+
+const getById = async (id) => {
+  return await User.findByPk(id);
```

Comment:
The `await` keyword is unnecessary here since the function is returning the promise directly. You can remove `await` and simply return the result of `User.findByPk(id);`.